### PR TITLE
Fix punctuation across the wiki

### PR DESCRIPTION
**Pull request by Calebtheaa**
Found more missing periods, (basically the whole wiki's missing periods)

### CODE REVIEW

File: wiki/Skinning/Interface/en.md [hunk 1]
```diff
@@ -1171,8 +1171,8 @@ Notes:
 Notes:
 
 - `scorebar-marker.png` has higher priority.
-- this element represents the "warning" zone
-- This element is not used in [osu!mania](/wiki/Game_mode/osu!mania)
+- This element represents the "warning" zone.
+- This element is not used in [osu!mania](/wiki/Game_mode/osu!mania).
 - Y-position at 16; x-position is placed at the end of the cropped `scorebar-colour.png`
```
Comment (peppy) [inline wiki/Skinning/Interface/en.md:1176]: any reason for not adding on this line too?

**Comment by TPGPL** [comment]
@Calebtheaa are you still working on this PR, or is it ready to be reviewed in the current state?

**Comment by Calebtheaa** [comment]
You can delete it.. I moved on from this.

**Comment by Calebtheaa** [comment]
You can delete it, i oved on from this.

**Pull request closed without merging**